ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -341,26 +341,25 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )build,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
     branches:
     - ^master$
     - ^master-
     cluster: build03
-    context: ci/prow/ci-index
+    context: ci/prow/ci-index-wmco-bundle
```

Comment:
Is the `ci-index` getting adding by `make jobs`?

Reply:
Yes, all changes to the jobs files are being made by `make jobs`